app/assets/stylesheets/overrides.css.less

```diff
@@ -320,3 +320,8 @@ html, body {
 .hide {
   display: none;
 }
+
+.panel-footer{
+  height: 80px;
```

Comment:
This should be in ems rather than px if possible, so that it's accessible for people with extreme default font sizes (e.g. those with bad eyesight and large fonts, those on small screens and tiny fonts).